lib/jetstream/api/stream.ex

```diff
@@ -375,16 +379,16 @@ defmodule Jetstream.API.Stream do
       iex> {:ok, %{total: _, offset: 0, limit: 1024, streams: _}} = Jetstream.API.Stream.list(:gnat)
 
   """
-  @spec list(conn :: Gnat.t(), params :: [{:offset, non_neg_integer()}, {:subject, binary()}]) ::
+  @spec list(conn :: Gnat.t(), domain :: nil | binary(), params :: [{:offset, non_neg_integer()}, {:subject, binary()}]) ::
           {:ok, streams()} | {:error, term()}
-  def list(conn, params \\ []) do
+  def list(conn, domain \\ nil, params \\ []) do
```

Comment:
Maybe we can also pass the domain in the `params` option list here as well? Avoid another breaking API change?

Reply:
Good idea, i've incorporated this and the other requested changes in a new commit. Thanks for reviewing!